src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -318,8 +318,8 @@ private function detectMappingType(string $directory, ContainerBuilder $containe
                 break;
             }
             if (
-                preg_match('/^ \* @.*'.$quotedMappingObjectName.'\b/m', $content) ||
-                preg_match('/^ \* @.*Embeddable\b/m', $content)
+                preg_match('/^( \*|\/\*\*) @.*'.$quotedMappingObjectName.'\b/m', $content) ||
```

Comment:
Maybe use a non-capturing group here and below?

```suggestion
                preg_match('/^(?: \*|\/\*\*) @.*'.$quotedMappingObjectName.'\b/m', $content) ||
```